golem-shard-manager/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:bookworm-slim
+
+ARG TARGETPLATFORM
```

Comment:
I strongly recommmend not to abstract things here

Reply:
Because a change for ARM will result in having to test the build for amd64.
I understand DRY principle, however, sometimes its ok to have simply two folders and have the most simplest Dockerfile. DevOps can quickly get convoluted and nasty.

Reply:
There might be requirements of additional installations for a target platform. I am not 100 percent sure. Unless we are so sure this will never change, then its ok. Good that base image is apparently same.

Reply:
This is also going with the assumption, our current docker build strategies is going to be permanent (i.e, building binaries outside targetting a platform).

Cross compilation in Rust is not that straight forward, and may be in a few months, we will have to revisit that as well, breaking this Dockerfile anyway

Reply:
I would rather have a single dockerfile than more than one. If we have more than one, with almost the same content, we will always forget to update one of them and it's going to be annoyoing.

Reply:
everything what i seen till now is using that approach (one common dockerfile), 
for multi platform build , if there are needs for platform customisation, it should be possible to do stages based on given platform https://github.com/docker/buildx/issues/805#issuecomment-946478949

but sure we can try to find if it is possible to do it in different way ...

Reply:
Ok we are using standard solutions here, and I can now see why this approach you took. 👍